### DISCUSSION TITLE
How to use this repo

### DISCUSSION THREAD

**Discussion by E2-venkat** [Schwartzblat/WhatsAppPatcher#71] [Q&A]
I'm using mac mini m1 how to use this patch readme stuffs are not working python3 installed

**Comment by Schwartzblat** [comment]
Hi,
What's the error? It supposed to work fine on macOS too.

**Reply by E2-venkat** [reply]
File doesn't exists or required reading permissions getting this

**Reply by Schwartzblat** [reply]
Send the the traceback.

**Comment by E2-venkat** [comment]
for python 3 use 3 at the end example pip3 , python3
